protoc-gen-gogo/generator/generator.go

```diff
@@ -1183,12 +1183,26 @@ func (g *Generator) generateHeader() {
 // The path is a comma-separated list of integers.
 // See descriptor.proto for its format.
 func (g *Generator) PrintComments(path string) {
-	if loc, ok := g.file.comments[path]; ok {
-		text := strings.TrimSuffix(loc.GetLeadingComments(), "\n")
-		for _, line := range strings.Split(text, "\n") {
-			g.P("// ", strings.TrimPrefix(line, " "))
-		}
+	text := g.Comments(path)
```

Comment:
I agree with it. I reverted it at https://github.com/ceram1/protobuf/commit/54e5631f0de9fbab0fdf83f008494a5be49dbb0e